store-cli.go

```diff
@@ -259,8 +272,12 @@ func main() {
 				}
 				scope := strings.ToLower(c.String("scope"))
 				storeType := strings.ToLower(c.String("type"))
+				timeout, err := getTimeout(c.Int("timeout"), "SD_STORE_CLI_DOWNLOAD_HTTP_TIMEOUT", 60)
```

Comment:
I think it would be better to define the default timeout time as a constant. Same for the others.

Reply:
That's sounds nice. I will fix it.

Reply:
Your comments were very helpful. I've updated the code accordingly. Please review it again.